cmd/main.go

```diff
@@ -17,23 +17,20 @@ import (
 )
 
 func main() {
-	// get org and repo flags
-	repoStr := flag.String("repository", "", "owner and repository to fetch build logs, e.g. ossf/scorecard")
-	digestStr := flag.String("digest", "", "sha256 digest of the input binary being produced")
-	flag.Parse()
-
-	if *repoStr == "" {
-		flag.Usage()
-		return
+	// Check for org and repo env variables
+	repository, ok := os.LookupEnv("INPUT_REPOSITORY")
+	if !ok {
+		fmt.Fprintln(os.Stderr, "Environment variable INPUT_REPOSITORY not present")
+		os.Exit(1)
 	}
-
-	if *digestStr == "" {
-		flag.Usage()
-		return
+	digest, ok := os.LookupEnv("INPUT_DIGEST")
+	if !ok {
+		fmt.Fprintln(os.Stderr, "Environment variable INPUT_DIGEST not present")
```

Comment:
shall we use panic("") everywhere? I'm not sure what the standard practice is tbh